phpBB/includes/acp/acp_extensions.php

```diff
@@ -98,7 +98,7 @@ function main()
 			catch (exception_interface $e)
 			{
 				$message = call_user_func_array(array($this->user, 'lang'), array_merge(array($e->getMessage()), $e->get_parameters()));
-				trigger_error($message, E_USER_WARNING);
+				trigger_error($message . adm_back_link($this->u_action), E_USER_WARNING);
```

Comment:
Consistent with all other cases of `trigger_error` on metadata failure, include a back link